pkgs/development/tools/ilspycmd/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, powershell
+, autoSignDarwinBinariesHook
+, glibcLocales
+}:
+buildDotnetModule rec {
+  pname = "ilspycmd";
+  version = "8.0";
+
+  src = fetchFromGitHub {
+    owner = "icsharpcode";
+    repo = "ILSpy";
+    rev = "v${version}";
+    hash = "sha256-ERBYXgpBRXISfqBSBEydEQuD/5T1dvJ+wNg2U5pKip4=";
+  };
+
+  nativeBuildInputs = [
+    powershell
+  ] ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) [ autoSignDarwinBinariesHook ];
```

```suggestion
  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ autoSignDarwinBinariesHook ];
```
same thing but shorter